lib/matplotlib/cm.py

```diff
@@ -147,13 +155,36 @@ def get_cmap(name=None, lut=None):
         return cmap_d[name]._resample(lut)
 
 
+def unregister_cmap(name):
+    """
+    Remove a colormap recognized by :func:`get_cmap`.
+
+    You may not remove built in color maps.
+
+    If the named color map is not registered, returns with no error, raises
+    if you try to de-register a default color map.
+
+    Raises
+    ------
+    ValueError
+       If you try to de-register a default build-in color map.
+    """
```

Comment:
~~~
    Notes
    -----
    Colormap names are currently a shared namespace that may be used by
    multiple packages. Use `unregister_cmap` only if you know you have
    registered that name before. In particular, do not unregister just in case
    to clean the name before registering a new colormap.
~~~